### PR TITLE
Um/asc 18934/post impression

### PR DESCRIPTION
**Pull request by umareko**

### CODE REVIEW

File: lib/presentation/screen/dashboard/dashboar_screen.dart
Comment (bnaing) [inline lib/presentation/screen/dashboard/dashboar_screen.dart:1]: finally i found something that i can make comment, please fix typo in file name 🤣

**Review by bnaing — CHANGES_REQUESTED** [review]

File: lib/presentation/screen/dashboard/dashboar_screen.dart
Comment (umareko) [reply lib/presentation/screen/dashboard/dashboar_screen.dart:1]: LoL, this typo is as old as the flutter sample app 😅  . I just noticed it now.  Will change it.   😂😂😂😂

Comment (umareko) [reply lib/presentation/screen/dashboard/dashboar_screen.dart:1]: Done @bnaing

**Review by bnaing — APPROVED** [review]
LGTM

**Pull request merged**